### DISCUSSION TITLE
bookmark numeration

### DISCUSSION THREAD

**Discussion by ifsergiy** [PaulWoitaschek/Voice#1891] [Ideas]
please make some kind of bookmark numeration  for more comfortable use

![Screenshot_56](https://user-images.githubusercontent.com/17888446/87531441-629de680-c69a-11ea-807b-ce81658e93a1.png)

**Comment by kartik-karz** [comment]
The bookmark takes the name of the chapter by default (assuming it's provided in the audio file). It also overrides this if you enter a bookmark name so I don't think this matters much. You could type numbers instead and it will work the way you wanted. Having a counter to append to the chapter name would probably be better choice.
